tests/ro/creator/data/roc-02-emobon-observatory.yml

```
@@ -0,0 +1,70 @@
+uses: emobon-observatory  # directive to drive specific
+base:  !resolve "{domain}/{repo}"
```

YAMLlint (1.35.1)

[error] too many spaces after colon

(colons)